presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -5885,13 +5885,13 @@ public void testCollectColumnStatisticsOnCreateTable()
         // non existing partition
         assertQuery(format("SHOW STATS FOR (SELECT * FROM %s WHERE p_varchar = 'p3')", tableName),
                 "SELECT * FROM VALUES " +
-                        "('c_boolean', null, 0E0, 0E0, null, null, null), " +
-                        "('c_bigint', null, 0E0, 0E0, null, null, null), " +
-                        "('c_double', null, 0E0, 0E0, null, null, null), " +
-                        "('c_timestamp', null, 0E0, 0E0, null, null, null), " +
-                        "('c_varchar', 0E0, 0E0, 0E0, null, null, null), " +
-                        "('c_varbinary', null, 0E0, 0E0, null, null, null), " +
-                        "('p_varchar', 0E0, 0E0, 0E0, null, null, null), " +
+                        "('c_boolean', 0, 0, 1, null, null, null), " +
+                        "('c_bigint', 0, 0, 1, null, null, null), " +
+                        "('c_double', 0, 0, 1, null, null, null), " +
+                        "('c_timestamp', 0, 0, 1, null, null, null), " +
+                        "('c_varchar', 0, 0, 1, null, null, null), " +
+                        "('c_varbinary', 0, 0, 1, null, null, null), " +
+                        "('p_varchar', 0, 0, 1, null, null, null), " +
```

Comment:
there is a bunch of redudant changes: 0e0 -> 0

there are also some meaningful changes:
- 0e0 -> 1
- null -> 0

ideally let's avoid unnnecessary changes within the "logic change" commit
more importantly -- why the meaningful changes?

Reply:
I am going to extract redundant changes to separate commit.

Meaningful changes are after stats normalization, when there is no rows, then we also know how much nulls, ndv or data size is there.

Reply:
WRT meaningful changes. Notice it is not the only thing. For example when min and max are equal, then ndv will be equal to 1.